Lib/test/test_clinic.py

```diff
@@ -1386,7 +1387,7 @@ def _do_test(self, *args, expect_success=True):
         ) as proc:
             proc.wait()
             if expect_success and proc.returncode:
-                self.fail("".join(proc.stderr))
+                self.fail("".join(itertools.chain(proc.stdout, proc.stderr)))
```

Comment:
Since clinic never writes to stderr, we should be fine with this, so we don't need to add a new import. What do you think?

```suggestion
                self.fail("".join(proc.stdout))
```

Reply:
Not sure -- what if a `ValueError` doesn't get caught by clinic, and causes the test to fail for some unexpected reason?

We could do it without the `itertools` import like this, but it feels slightly less idiomatic to me:

```suggestion
                self.fail("".join([*proc.stdout, *proc.stderr]))
```

Reply:
> [...] what if a ValueError (or whatever) doesn't get caught by clinic (due to a bug), and causes the test to fail for some unexpected reason, with the exception message being written to stderr?

Good point.

> Though being fully idiomatic doesn't really matter too much here, since this is a test, after all! Then again, an extra import shouldn't matter too much here either ;)

Given this choice, I'll easily go for the non-idiomatic approach :)